lxd/operations.go

```diff
@@ -44,6 +47,69 @@ var operationWebsocket = APIEndpoint{
 	Get: APIEndpointAction{Handler: operationWebsocketGet, AllowUntrusted: true},
 }
 
+func waitForOperations(s *state.State, force chan bool) {
```

Comment:
@monstermunchkin would `force` be clearer named `cancel` or `forceCancel` as otherwise, without comment its not clear what that channel actually forces.

Reply:
`chCancel` would also make it clearer that this is not a variable you can just check